src/cast-engine.js

```diff
@@ -261,15 +258,17 @@ class CastEngine extends FakeEventTarget {
   }
 
   get duration(): number {
-    if (this.isLive()) {
-      const range = this._playerManager.getLiveSeekableRange();
-      if (range) {
-        return range.end - this.getStartTimeOfDvrWindow();
-      }
-    }
     return this._playerManager.getDurationSec();
   }
 
+  get liveDuration(): number {
+    const range = this._playerManager.getLiveSeekableRange();
+    if (range) {
+      return range.end;
```

Comment:
dont you need to reduce the getStartTimeOfDvrWindow? like was before in `duration`?